src/models/service/transactions/details.rs

```diff
@@ -37,6 +37,7 @@ pub struct MultisigExecutionDetails {
 pub struct MultisigConfirmation {
     pub signer: String,
     pub signature: Option<String>,
+    pub submission_time: Option<i64>,
```

Comment:
this should not be optional

Reply:
so I would expect this https://github.com/gnosis/safe-client-gateway/blob/ca68312297009dfe934043de0d96e674a41d3779/src/models/backend/transactions.rs#L86 also to be not optional

Reply:
It is optional from the backend: https://github.com/gnosis/safe-client-gateway/blob/8491e453ea2e367300bd4098f87a5025e14d4060/src/models/backend/transactions.rs#L86

Reply:
`confirmations` is a string in the `Multisig` variant in the docs 🤔 

https://safe-transaction.rinkeby.gnosis.io/

Reply:
submission_date will always be present. See https://github.com/gnosis/safe-transaction-service/blob/master/safe_transaction_service/history/serializers.py#L267 (`created` is a database field that is set upon db object creation and will always be present)

Reply:
alright, will remove the `Option<>` wrapper.